test/floki/deep_text_test.exs

```diff
@@ -9,6 +9,12 @@ defmodule Floki.DeepTextTest do
     assert Floki.DeepText.get(node) == "Google"
   end
 
+  test "text from simple node with separator" do
+    node = {"a", [], ["Google"]}
+
+    assert Floki.DeepText.get(node, " ") == " Google"
```

Comment:
I'm not sure if it's useful to add a space in the beginning of the result. Seems unnecessary.

I believe the separator should work like `Enum.join/2`.
E.g.:

```elixir
Enum.join(["Hello", "World"], " ")
=> "Hello World"
```

@rochdi What do you think?